Newsletter2Go/Export/Model/Config/Source/IntegrationCreateButton.php

```diff
@@ -7,35 +7,38 @@
 
 class IntegrationCreateButton extends Field
 {
-
-    const NEWSLETTER2GO_CONNECT_URL = "https://ui.newsletter2go.com/integrations/connect/MAG2/?version=4000&token=<token>&url=";
+    const NEWSLETTER2GO_CONNECT_URL = 'https://ui.newsletter2go.com/integrations/connect/MAG2/?version=4000&token=<token>&url=';
 
     /**
      * Retrieve element HTML markup
      *
      * @param AbstractElement $element
+     *
      * @return string
      */
     protected function _getElementHtml(AbstractElement $element)
     {
         $element->setType('button');
         $element->setData('value', 'Connect to Newsletter2Go');
 
-        if (!$this->_scopeConfig->getValue('newsletter2go/general/token')) {
-            $element->setDisabled('true');
+        if (!$this->_scopeConfig->getValue('newsletter_go/general/token')) {
+            $element->setData('disabled', 'true');
```

Comment:
There's a `__set` method that handles `setDisabled` and calls `setData`... I decided to go with the `setData` directly since:
1. we get type-hinting and we can see what will happen
2. a lot of code already makes use of `setData` instead of the nice-looking (but deceitful) magic calls.